.gitignore

```diff
@@ -269,3 +269,7 @@ binaries/
 
 # Do not include launch settings
 launchSettings.json
+
+.dotnet/
+artifacts/
+/.packages
```

Comment:
You should also add folders from eng/common that you don't need/care about, like `cross` , `internal` and `native` those are directories that you will not use and I don't imagine corefxlab using it, so I think is better to exclude them and if needed we can add them later on.